Gemfile_rack_2

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gemspec
+
+gem 'byebug'
+gem 'rack', '>= 2.0.0.rc1'
```

Comment:
`>=` is not a good constraint (at least, without an accompanying `<`). will include rack 3 when that comes out (in 7 years ... if the time between 1.0.0 and 2.0.0 is anything to go by). `~> 2.0` is good.